VisCap.R

```diff
@@ -566,7 +566,7 @@ mat.all[which(mat.all==0)]=0.00001
 interval_lookup <- load_interval_names(interval_list_dir, interval_file_pattern)
 
 # Sort matrix by genome coordinates found in rownames
-chroms <- c(1:22,"X","Y", "MT", "M")
+chroms <- c(paste("chr",1:22,sep=""),"chrX","chrY", "chrMT", "chrM")}else{chroms <- c(1:22,"X","Y", "MT", "M")}

```

Comment:
Can you fix this - there is no if statement

Reply:
Thanks to your reviewing, I fix the if statement